src/Illuminate/Support/Arr.php

```diff
@@ -233,7 +234,8 @@ public static function get($array, $key, $default = null)
         }
 
         foreach (explode('.', $key) as $segment) {
-            if (! is_array($array) || ! array_key_exists($segment, $array)) {
+            if ((! is_array($array) || ! array_key_exists($segment, $array)) &&
```

Comment:
Wouldn't that introduce a false positive then? If I make my default value let's say `false` and the the array key exists, but is `null`, with `isset` I won't get `null`, but `false` as a result instead.